scripts/test/js_test.sh

```diff
@@ -7,6 +7,9 @@ then
 elif [[ ! -z "$CODECOV" ]]
 then
     export CMD="node ./node_modules/nyc/bin/nyc.js --reporter=lcovonly -R spec mocha"
+elif [[ ! -z "$WATCH" ]]
+then
+    export CMD="node ./node_modules/.bin/mocha --watch"
```

Comment:
yeah i think those should be consistent and the presence of `$WATCH` should just tack on the `--watch` parameter at the end of the command